torch/fx/passes/infra/partitioner.py

```diff
@@ -268,7 +268,7 @@ def fuse_partitions(self, partitions: List[Partition], prefix: str = "fused_") -
         # fuse_by_partitions expects partitions in List[List[Node]]: [ [node0, node1], [node2, node3] ]
```

Comment:
This comment needs updating I guess?

Reply:
It should be updated. I will submit other PR due to this PR has been merged.

Reply:
See https://github.com/pytorch/pytorch/pull/135662.